### ISSUE TITLE
[Feature]: Route guids should be valid UUIDs

### ISSUE THREAD

**Issue by danail-branekov** [cloudfoundry/korifi#2330]
### Blockers/Dependencies

No response

### Background

As a CF API client I want routes to have guids that are valid UUIDs so that existing client implementations can safely treat route guids as UUIDs

### Acceptance Criteria

GIVEN Korifi is deployed
WHEN I create a route by

```
curl "https://api.example.org/v3/routes" \
  -X POST \
  -H "Authorization: bearer [token]" \
  -H "Content-type: application/json" \
  -d '{
    "host": "a-hostname",
    "path": "/some_path",
    "port": 6666,
    "relationships": {
      "domain": {
        "data": { "guid": "domain-guid" }
      },
      "space": {
        "data": { "guid": "space-guid" }
      }
    }
  }'
```
THEN I get a response similar to

```
HTTP/1.1 201 Created
Content-Type: application/json

{
  "guid": "cbad697f-cac1-48f4-9017-ac08f39dfb31",
  ...
}
```

### Dev Notes
We are currently [setting the `cf-route-` prefix](https://github.com/cloudfoundry/korifi/blob/221eef73c55f284f595dc4fe2ac5a7e6fdf5fe15/api/repositories/route_repository.go#L131) when generating the routes GUID, can we remove it?

**Comment by georgethebeatle** [comment]
We did a short spike on the topic. Here is the branch: https://github.com/cloudfoundry/korifi/commit/5d9fbbdf55b7d61761665a4036988dea9702e3d1

TL;DR we can push the prefixing of the guids of some resources into a webhook so that korifi can choose to switch it on and off.
